src/test/ui/borrowck/borrowck-mut-borrow-linear-errors.ast.nll.stderr

```diff
@@ -13,7 +13,7 @@ error[E0499]: cannot borrow `x` as mutable more than once at a time
   --> $DIR/borrowck-mut-borrow-linear-errors.rs:15:30
    |
 LL |             1 => { addr.push(&mut x); } //[ast]~ ERROR [E0499]
-   |                    ---- first borrow used here, in later iteration of loop
```

Comment:
The two changes in this files also happened in AST and we already approved that.

Reply:
I don't see an analogous message in 
https://github.com/rust-lang/rust/blob/06ec1a422a5ff147c3a09af7eb80db77459e797b/src/test/ui/borrowck/borrowck-mut-borrow-linear-errors.ast.stderr#L4-L20

Were you referring to a different file when you said "also happened in AST" ? Or do you mean that the above diagnostic is analogous to what NLL is being changed to produced above?

(To be clear, I think AST is currently producing a third totally different diagnostic message right now for this case...)

Reply:
(note in particular that the AST-borrowck error message is highlighting a use of `x` in the second match arm, while NLL is highlighting a use of `addr` in the first match arm.)

I'm not saying I object to the change. I'm just trying to understand the comment you have written here on github about it.